src/ansible_navigator/cli.py

```diff
@@ -125,7 +125,7 @@ def run(args: ApplicationConfiguration) -> ActionReturn:
     if args.mode == "stdout":
         try:
             result = run_action_stdout(args.app.replace("-", "_"), args)
-            return result
+            return result  # noqa: TRY300
```

Comment:
Not ok to use noqa for this, read https://docs.astral.sh/ruff/rules/try-consider-else/ gives an example that should work just fine.